R/annotation-ticks.R

```diff
@@ -0,0 +1,242 @@
+#' @title Annotation: tick marks
+#' @description  This annotation adds tick marks to an axis
+#' @export
+#' @inheritParams annotation_logticks
+#' @param ticks_per_base integer, number of minor ticks between each pair of major ticks, Default: base-1
+#' @param delog boolean, if an idenity transformation is needed use set to TRUE, Default: FALSE
+#' @examples
+#' 
+#' minorUtil=function(scale,side,ticks){
+#'   data.frame(
+#'     base =c('log'=exp(1),'log10'=10,'identity'=10)[scale],
+#'     delog=scale=='identity',
+#'     ticks_per_base=ticks+1,
+#'     stringsAsFactors = FALSE
+#'   )
+#' }
+#' 
+#' minorTicks<-c('blt')
+#' minorTickNum<-c(10,3,4)
+#' yScale<-'identity'
+#' xScale<-'log10'
+#' 
+#' tickMat=mapply(minorUtil,
```

Comment:
Is it possible to simplify these examples? It would be great if you could show a simple example without having to use mapply, plyr, or dplyr.